docs/local-dind-tutorial.md

```diff
@@ -135,21 +138,46 @@ To access the TiDB cluster, use `kubectl port-forward` to expose the services to
         * Default username: admin
         * Default password: admin
 
+- Permanent remote access
+
+    Although this is a very simple demo cluster and not suitable for any serius usage, it may be useful if it can be accessed remotely without `kubectl port-forward`, which may require an open terminal.
+
+    By default, TiDB, Prometheus, and Grafana are exposed as `NodePort` Services, so it's possible to setup reverse proxy for them.
+
+    1. You can find their listing port numbers by:
+
+        ```sh
+        $ kubectl get service -n tidb | grep NodePort
+        demo-grafana      NodePort    10.111.80.73     <none>        3000:32503/TCP                   1m
+        demo-prometheus   NodePort    10.104.97.84     <none>        9090:32448/TCP                   1m
+        demo-tidb         NodePort    10.102.165.13    <none>        4000:32714/TCP,10080:32680/TCP   1m
+        ```
+
+        In this sample output, the ports are: 32503 for Grafana, 32448 for Prometheus, and 32714 for TiDB.
+
+    2. Find host IP address of the cluster.
+
+        DinD is a k8s cluster running inside docker containers, so Services expose ports to the containers' address, instead of the real host machine. We can find IP addresses of docker containers by `kubectl get nodes -o yaml | grep address`.
+
+    3. Setup reverse proxy
+
+        Either (or all) of the container IPs can be used as upstream for reverse proxy. You may use any reverse proxy server that supports TCP (for TiDB) or HTTP (for Grafana and Prometheus) to provide remote access. HAProxy and nginx are two common choises.
+
 ## Scale the TiDB cluster
 
-You can scale out or scale in the TiDB cluster simply by modifying the number of `replicas`.
+You can scale up or scale down the TiDB cluster simply by modifying the number of `replicas`.
```

Comment:
We use scale in and scale out for horizontal scale, and scale up and scale down for vertical scale.